src/inference/include/openvino/runtime/core.hpp

```diff
@@ -519,6 +519,31 @@ class OPENVINO_RUNTIME_API Core {
      */
     Any get_property(const std::string& device_name, const std::string& name, const AnyMap& arguments) const;
 
+    /**
+     * @brief Gets properties related to core behaviour.
+     *
+     * The method extracts information that can be set via the set_property method.
+     *
+     * @param name  Property name.
+     * @return Value of a property corresponding to the property name.
+     */
+    Any get_property(const std::string& name) const {
+        return get_property(std::string(), name);
+    }
+
+    /**
+     * @brief Gets properties related to core behaviour.
+     *
+     * The method extracts information that can be set via the set_property method.
+     *
+     * @param name  Property name.
+     * @param arguments  Additional arguments to get a property.
+     * @return Value of a property corresponding to the property name.
+     */
+    Any get_property(const std::string& name, const AnyMap& arguments) const {
```

Comment:
Do we need this method?

Reply:
Actually, I am not sure, just stay aligned with the others.

Reply:
we don't need this